packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -241,6 +253,57 @@ export const createRDSCluster = async (config: RDSConfig): Promise<ClusterInfo>
   }
 };
 
+/**
+ * Accesses the local RDS Aurora serverless V2 cluster with one DB instance using the given input configuration.
+ * @param config Configuration of the database cluster.
+ * @returns EndPoint address, port and database name of the accessed RDS cluster.
+ */
+export const useRDSCluster = async (config: RDSConfig): Promise<ClusterInfo> => {
```

Comment:
I'm confused about this method -- you named it "use" but it appears to only be gathering information about the cluster, not establishing connections or anything. What is the actual purpose, and what name would more accurately reflect its intent? Similarly, the comment above "accesses the local cluster..." isn't helpful. Why is it accessing the cluster--to validate? To retrieve? To establish a long-term connection?